Upgrade to Sf3.4

pdias opened the pull request:
Make services public by default

pdias committed:
Upgrade to Sf3.4

Make services public by default.

pdias committed:
Upgrade to Sf3.4

Make services public by default

pdias committed:
Upgrade to Sf3.4

Make services public by default

pdias committed:
Upgrade to Sf3.4

Make services public by default

pdias committed:
Upgrade to Sf3.4

Make services public by default

pdias committed:
Upgrade to Sf3.4

Make services public by default

pdias committed:
Upgrade to Sf3.4

Make services public by default

pdias committed:
Upgrade to Sf3.4

Make services public by default

pdias committed:
Upgrade to Sf3.4

Make services public by default

pdias committed:
Upgrade to Sf3.4

Make services public by default

pdias committed:
Upgrade to Sf3.4

Make services public by default

pdias committed:
Upgrade to Sf3.4

Make services public by default

pdias committed:
Upgrade to Sf3.4

Make services public by default

pdias committed:
Upgrade to Sf3.4

Make services public by default

pdias committed:
Upgrade to Sf3.4

Make services public by default

pdias committed:
Upgrade to Sf3.4

Make services public by default

pdias committed:
Upgrade to Sf3.4

Make services public by default

pdias committed:
Upgrade to Sf3.4

Make services public by default

pdias committed:
Upgrade to Sf3.4

Make services public by default

pdias committed:
Upgrade to Sf3.4

Make services public by default

pdias committed:
Upgrade to Sf3.4

Make services public by default

pdias committed:
Upgrade to Sf3.4

Make services public by default

pdias committed:
Upgrade to Sf3.4

Make services public by default

pdias committed:
Upgrade to Sf3.4

Make services public by default

pdias committed:
Upgrade to Sf3.4

Make services public by default

pdias committed:
Upgrade to Sf3.4

Make services public by default

pdias committed:
Upgrade to Sf3.4

Make services public by default

pdias committed:
Upgrade to Sf3.4

Make services public by default

pdias committed:
Upgrade to Sf3.4

Make services public by default

pdias committed:
Upgrade to Sf3.4

Make services public by default

pdias committed:
Upgrade to Sf3.4

Make services public by default

pdias committed:
Upgrade to Sf3.4

Make services public by default

pdias committed:
Upgrade to Sf3.4

Make services public by default

pdias committed:
Upgrade to Sf3.4

Make services public by default

pdias committed:
Update to Sf3.4

Make services public by default

pdias committed:
Upgrade to Sf3.4

Make services public by default

pdias committed:
Upgrade to Sf3.4

Make services public by default

pdias committed:
Update to Sf3.4

Make services public by default

pdias committed:
Update to Sf3.4

Make services public by default

pdias committed:
Update to Sf3.4

Make services public by default

pdias committed:
Update to Sf3.4

Make services public by default

pdias committed:
Update to Sf3.4

Make services public by default

pdias committed:
Update to Sf3.4

Make services public by default

pdias committed:
Upgrade to Sf3.4

Make services public by default

Jean-Beru commented on Resources/config/assetic.xml:
```diff
@@ -25,6 +25,9 @@
 
     <services>
+        
+        <defaults public="true" />
```
`<defaults public="true" />` is not compatible with SF2.3. But set each service can be set as public.

hackzilla commented:
My gut says it's because PHP Unit is a newer version.

```
Fatal error: Class 'PHPUnit_Framework_TestCase'
```

I believe it changed to \PHPUnit\Framework\TestCase

Nek- commented:
This doesn't fix everything for 3.4. Also, it needs to be squash.

ajtis commented:
+1

pdias committed:
Set service public 

<defaults public="true" /> is not compatible with SF2.3. 
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3. 
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Set service public

<defaults public="true" /> is not compatible with SF2.3.
But set each service can be set as public.

pdias committed:
Auto-registration of the command

Deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead.

pdias committed:
Auto-registration of the command 

Deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead.

pdias committed:
Update command.xml

pdias committed:
Update command.xml
